runtime/command-start/src/main/java/io/aklivity/zilla/runtime/command/start/internal/airline/ZillaStartCommand.java

```diff
@@ -51,7 +51,7 @@ public final class ZillaStartCommand extends ZillaCommand
     @Option(name = { "-c", "--config" },
             description = "Configuration location",
             hidden = true)
-    public URI configURL = Paths.get("zilla.json").toUri();
+    public String configFile;
```

Comment:
Why change from URL to file while adding the indirection through `zilla.properties`?
The configuration is located at a URL, for example we'll be supporting `http(s)` URLs soon too.